OBAKitCore/Network/RESTAPIService/RESTAPIService+GetAgencyAlerts.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  RESTAPIService+GetAgencyAlerts.swift
+//  OBAKitCore
+//
+//  Created by Alan Chu on 12/29/22.
+//
+
+extension RESTAPIService {
+    // MARK: - Regional Alerts
+    public nonisolated func getAlerts(agencies: [AgencyWithCoverage]) async throws -> [AgencyAlert] {
```

Comment:
This used to be in `MultiAgencyAlertsOperation.swift`, which was mostly boilerplate code for Operation.